#Expyriment 1.0.0 (Python 3.10.14), .xpe-file, coding: cp1252
#date: Wed Oct 08 2025 17:05:54
#sha1: ffae97
#modules: 
#display: size=(800, 600), window_mode=True, opengl=2
#os: uname_result(system='Windows', node='LAPTOP-909EKFKU', release='10', version='10.0.26100', machine='AMD64')
Time,Type,Event,Value,Detail,Detail2
644,stderr,received,'Traceback (most recent call last):'
644,stderr,received,'  File "C:\\Users\\USER\\OneDrive\\Desktop\\GITHUB\\cogsup-prog\\Assignments\\Week-4\\Exercises\\drawing_functions.py", line 78, in <module>'
644,stderr,received,'    present_for(stims, 500)'
644,stderr,received,'  File "C:\\Users\\USER\\OneDrive\\Desktop\\GITHUB\\cogsup-prog\\Assignments\\Week-4\\Exercises\\drawing_functions.py", line 37, in present_for'
645,stderr,received,'    dur = timed_draw(exp, stims) #call here the method the give the actual process of drawing'
645,stderr,received,'  File "C:\\Users\\USER\\OneDrive\\Desktop\\GITHUB\\cogsup-prog\\Assignments\\Week-4\\Exercises\\drawing_functions.py", line 26, in timed_draw'
645,stderr,received,'    time_before_draw = exp.clock.time'
645,stderr,received,"AttributeError: 'list' object has no attribute 'clock'"
